linter/spectral.yml

```diff
@@ -104,6 +104,25 @@ rules:
       field: "access-control-allow-origin"
     message: "The response of the `/openapi.json` resource should set the `access-control-allow-origin` header with value `*`"
 
+  #/core/doc-openapi-contact
+  # This rule exists in the base oas spectral linter set
+  info-contact: error
+
+  info-contact-fields-exist:
+    severity: error
+    given:
+      - "$.info.contact"
+    then:
+      function: schema
+      functionOptions:
+        schema:
+          type: object
```

Comment:
```suggestion
```
Krijg deze error als het geen object is:
`error  info-contact  Info object must have "contact" object.  info`
Wellicht overbodig om type te specificeren?